specifications/api/swagger.yaml

```
@@ -7379,6 +7393,47 @@ paths:
             schema: *ApiSharedWalletPatchData
       responses: *responsesPatchSharedWallet
 
+  /shared-wallets/{walletId}/transactions-construct:
+    post:
+      operationId: constructSharedTransaction
+      tags: ["Shared Transactions"]
+      summary: Construct
+      description: |
+        <p align="right">status: <strong>under development</strong></p>
+
+        Create a transaction to be signed from the shared wallet.
+
+        Works for the following fields:
+          - payments
+          - metadata
+          - validity_interval
+      parameters:
+        - *parametersWalletId
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema: *ApiConstructTransactionData
+      responses: *responsesConstructTransaction
+
+  /shared-wallets/{walletId}/transactions-decode:
+    post:
+      operationId: decodeSharedTransaction
+      tags: ["Shared Keys"]
```

```suggestion
      tags: ["Shared Transactions"]
```
(I think it rather fits into transactions)